Jenkinsfile

```diff
@@ -164,7 +166,7 @@ try {
         ws('workspace/build-cpu-openblas') {
           init_git()
           sh "ci/build.py --platform ubuntu_cpu /work/runtime_functions.sh build_ubuntu_cpu_openblas"
-          pack_lib('cpu')
+          pack_lib('cpu', mx_dist_lib)
```

Comment:
Sorry, that's the wrong approach :/ Could you please extend ```mx_lib``` instead of adding ```mx_dist_lib``` and create a new PR @rahul003?